spectree/plugins/flask_plugin.py

```diff
@@ -1,3 +1,4 @@
+from flask import current_app
```

Comment:
With this import, users that are using `falcon` or `starlette` will need to install `flask` as a dependency.

Reply:
According to [PEP8 imports](https://www.python.org/dev/peps/pep-0008/#imports):

> Imports are always put at the top of the file, just after any module comments and docstrings, and before module globals and constants.

I agree that this should be the recommended way.

So I'm thinking maybe we should let the users import plugins they need instead of the library imports everything.

The interface may look like this:

```py
api = SpecTree(FlaskPlugin)
```

This should be more intuitive than a string.

Reply:
I see that now. It didn't cross my mind why imports were withing the function. I'll fix that right away.

And I definitely like the later approach where uses passes plugin class to SpecTree init.